pkgs/games/freeciv/default.nix

```diff
@@ -1,6 +1,6 @@
 { lib, stdenv, fetchFromGitHub, autoreconfHook, lua5_3, pkg-config, python3
-, zlib, bzip2, curl, xz, gettext, libiconv
-, sdlClient ? true, SDL, SDL_mixer, SDL_image, SDL_ttf, SDL_gfx, freetype, fluidsynth
+, zlib, bzip2, curl, xz, gettext, libiconv, icu
+, sdlClient ? true, SDL2, SDL2_mixer, SDL2_image, SDL2_ttf, SDL2_gfx, freetype, fluidsynth
```

Comment:
The sdl2 client seem to be marked as experimental on the website, in fact it's quite buggy on sway. Maybe it's better to enable the gtk3 client instead or both of them simultaneously?

Reply:
none of the clients are enabled simultaneously, I think?
"freeciv" is sdl2 (formerly sdl), "freeciv_gtk" is gtk3, "freeciv_qt" is qt5, all different derivations, just as it was before.

Did the sdl client work well before they moved to sdl2?  I wouldn't know, I have no idea who uses that or why it even exists.

Reply:
> none of the clients are enabled simultaneously, I think? "freeciv" is sdl2 (formerly sdl), "freeciv_gtk" is gtk3, "freeciv_qt" is qt5, all different derivations, just as it was before.
> 
> Did the sdl client work well before they moved to sdl2? I wouldn't know, I have no idea who uses that or why it even exists.

I just think that the gtk3 client should be enabled by default

Reply:
I agree (that's the one I use), but:

- the gtk3 client is also pretty buggy
- I don't think changing defaults is in scope of a lowly upgrade PR

Reply:
> I agree (that's the one I use), but:
> 
>     * the gtk3 client is also pretty buggy
> 
>     * I don't think changing defaults is in scope of a lowly upgrade PR

This pr is already changing the default, from the sdl one to the sdl2 one(that is experimental).

Reply:
it's the upstream who pulled that rug, but you have a point.